modules/library/src/main/scala/zio/elasticsearch/ElasticPrimitive.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2022 LambdaWorks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.elasticsearch
+
+import zio.json.ast.Json
+import zio.json.ast.Json.{Num, Str}
+
+object ElasticPrimitive {
```

Comment:
Can you extract the `sealed trait` above, and provide `ElasticPrimitive` with implicit under it?